taxi/models.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=100, unique=True)
+    country = models.CharField(max_length=100)
+
+    class Meta:
+        verbose_name_plural = "All manufacturers"

```

Comment:
`verbose_name_plural` is a plural form of model name
https://docs.djangoproject.com/en/4.1/ref/models/options/#verbose-name-plural